cdap-app-templates/cdap-program-report/cdap-program-report-base/src/main/java/co/cask/cdap/report/main/RunMetaFileManager.java

```diff
@@ -41,24 +42,30 @@
   private static final Long DEFAULT_MAX_FILE_OPEN_DURATION = TimeUnit.HOURS.toMillis(6);
   private static final String SYNC_INTERVAL = "file.sync.interval.bytes";
   private static final String MAX_FILE_SIZE_BYTES = "file.max.size.bytes";
-  private static final String MAX_FILE_OPEN_DURATION = "file.max.open.duration";
+  private static final String MAX_FILE_OPEN_DURATION_MILLIS = "file.max.open.duration.millis";
```

Comment:
It is not file open duration right? It is the sync interval?

Reply:
that is a separate property. `SYNC_INTERVAL_TIME_MILLIS` this change is an improvement in naming of an existing property to include time unit.